index.html

```diff
@@ -59,6 +59,164 @@ <h1 id="lets_connect">LET'S CONNECT</h1>
+                    <div class="projec_title">Tonic</div>
+                    <div class="frame">
+                        <img src="images/Frame 2.png" alt="">
+                    </div>
+                </div>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="tags">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <button class="action">See Project</button>
+            </div>
+        </div>
+        <div class="card">
+            <div class="snapshoot_wapper">
+                <img class="snapshoot" src="images/Snapshoot Portfolio (3).png" alt="">
+            </div>
+            <div class="left_block">
+                <div class="left_block_title">
+                    <h1 class="projec_title">Multi-Post <br>Stories</h1>
+                    <div class="frame">
+                        <img src="images/Frame 2.png" alt="">
+                    </div>
+                </div>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="tags">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <button class="action">See Project</button>
+            </div>
+        </div>
+    </section>
+    <!-- end of works section -->
+    <section class="about_myself">
+        <div class="top">
+            <h1>About<br> Myself</h1>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of
+                my work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                contact me.</p>
+            <div class="social_links">
+                <h1 id="lets_connect">LET'S CONNECT</h1>
+                <ul class="links">
+                    <li>
+                        <a href="#">
+                            <img src="/images/twitter.png" alt="">
+                        </a>
+                    </li>
+
+                    <li>
+                        <a href="#">
+                            <img src="/images/Linkedin.png" alt="">
+                        </a>
+                    </li>
+
+                    <li>
+                        <a href="#">
+                            <img src="/images/Vector@2x.png" alt="">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="/images/Github.png" alt="">
+                        </a>
+                    </li>
+
+                    <li>
+                        <a href="#">
+                            <img src="/images/crossfinger.png" alt="">
```

Comment:
- [ ] Make sure these links are visible :+1: , to stick to the Figma design :pray: 
- [ ] Also make sure that your links redirect the user to your personal social media accounts :wink:  

### Your design
![Screenshot from 2023-01-14 15-27-14 (1)](https://user-images.githubusercontent.com/80458199/212474648-c28c371c-757d-4a74-97e5-5fdf4086ecae.png)

### Figma design

![Screenshot from 2023-01-14 15-27-48 (1)](https://user-images.githubusercontent.com/80458199/212474671-97bfc805-fb90-4141-b5c0-758f1fcc2ffa.png)

Reply:
Hi Dear @iambenkis 

- I deployed my code both in [GitHub pages](https://zewdiemc.github.io/Portfolio/) and [Netlify](https://zewdieportfolio.netlify.app/). 
- However, the social media icons are visible in Netlify but not on GitHub pages. 
- The code also works as expected in my local machine.
- I would kindly like to know what the problem is with GitHub pages.

Reply:
- [ Optional]  In order to solve the issue on Github pages just add a dot to the icon link instead of   `/images/Vector2.png` for example use `./images/Vector2.png`